deploy/ansible/README.md

```diff
@@ -0,0 +1,205 @@
+# Grafana Operator | Ansible
+
+Ansible scripts for deploying and configuring the Grafana Operator.
+
+# Playbooks
+
+## grafana-operator-cluster-resources.yaml
+
+Playbook for deploying the cluster level resources required for deploying the Grafana Operator.
+
+### Required K8s Permisions 
+These permissions are reuqied by the K8s user who runs this playbook. Typically this would be a cluster administrator.
+
+* create CRDs
+* create ClusterRoles
+
+### Parameters
+
+| Parameter                                        | Choices / **Defaults** | Comments
+|--------------------------------------------------|------------------------|---------
+| k8s\_host                                        |                        | K8s API to run this playbook against
+| k8s\_validate\_certs                             | **True** / False       | Whether to validate K8s API certificate
+| k8s\_api\_token                                  |                        | K8s API token to authenticate with. Mutually exclusive with `k8s_username` and `k8s_password`.
```

Comment:
@itewk in the code, this parameter is called `k8s_api_key` and not `k8s_api_token`

Reply:
fixed.